config/controller.yaml

```diff
@@ -105,6 +105,8 @@ spec:
           value: config-leader-election
         - name: METRICS_DOMAIN
           value: tekton.dev/pipeline
+        securityContext:
+          runAsNonRoot: false
```

Comment:
Can we just run these as nonroot? There's nothing they need to do as root afaik.

Reply:
Yep -- totally agree. Trying out `runAsUser: 1001`